tasks/src/translation/data.ts

```diff
@@ -38,7 +38,7 @@ const taskData: TaskData = {
 	libraries: TASKS_MODEL_LIBRARIES["translation"],
 	metrics:   [
 		{
-			description: "Translation is evaluated on BLEU Score. BLEU score is calculated by counting the number of shared single or subsequent tokens between the generated sequence and the reference. Subsequent n tokens are called “n-grams”. Unigram refers to a single token while bi-gram refers to token pairs and n-grams refer to n subsequent tokens. The score ranges from 0 to 1, in which 1 means the translation perfectly matched and 0 did not match at all.",
+			description: "The BLEU metric is calculated by counting the number of shared single or subsequent tokens between the generated sequence and the reference. Subsequent n tokens are called “n-grams”. Unigram refers to a single token; bigram to token pairs; and n-grams to n subsequent tokens. The score ranges from 0 to 1, where 1 means the translation matched perfectly and 0 did not match at all.",
```

Comment:
I'd say
Translation is evaluated on BLEU Score. BLEU score is calculated by counting the number of shared single or subsequent tokens between the generated sequence and the reference. Subsequent n tokens are called “n-grams”. Unigram refers to a single token while bi-gram refers to token pairs and n-grams refer to n subsequent tokens. The score ranges from 0 to 1, where 1 means the translation perfectly matched and 0 did not match at all.

Reply:
I think the first sentence can be removed since it might be redundant (Translation is evaluated on BLEU Score.)

Reply:
Ok, then apply both your comments.